src/com/github/Kraken3/AFKPGC/LagScanner.java

```diff
@@ -147,20 +146,11 @@ public void run() {
 					} else {
 						stats.put(enttype.name(), entcost);
 					}
-					if (totalCost >= lagSourceThreshold) {
-						AFKPGC.debug("Chunk ", chunkId, " passed source threshold with tiles and entities");
+					if (totalCost >= lagSourceThreshold && !fullScan) {
 						break; // if we cross the threshold, don't keep adding.
 					}
 				}
 			}
-			StringBuffer sb = new StringBuffer();

```

Comment:
This is critical debug information and needs to stay. The other stuff was fine to remove. This lists the actual items that resulted in the lag computation -- it's vital I have that to tune the algorithm. In fact it needs to be improved to show the "count" of the items ... sigh. That way I can detect significant lag sources that haven't been configured at all, yet.